test_runner/regress/test_replication_lag.py

```diff
@@ -0,0 +1,56 @@
+import threading
+
+from fixtures.log_helper import log
+from fixtures.neon_fixtures import NeonEnv, PgBin, wait_replica_caughtup
+
+
+def test_replication_lag(neon_simple_env: NeonEnv, pg_bin: PgBin):
```

Comment:
Describing idea of the test (exercise standby under aggressive gc to test for 'page had been already garbage collected' errors and naming it accordingly would be nice.

Reply:
Comment is added